google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -518,6 +518,8 @@ private static Severity severityFor(Level level) {
         // SEVERE
       case 1000:
         return Severity.ERROR;
+      case Integer.MAX_VALUE:
+        return Severity.UNRECOGNIZED;
```

Comment:
nit: this method's original implementation is bad because it tries to use the internal knowledge about `class java.util.logging.Level` constants to make the conversion when it should actually substitute the Level's constants with the Severity's constants.
The change introduces an additional complexity by enforcing the "closed" range for "default" values which previously was bound only from the bottom (even despite the fact that the top boundary is set to the integer's maximal value).

Reply:
Changed this to ranges evaluation